CHANGELOG.md

```diff
@@ -1,10 +1,20 @@
-# v0.2.0
+# Changelog
+All notable changes to this project will be documented in this file.
 
-- Updated aws-sdk dependency to support IAM session credentials
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```

Comment:
`https` should be `http`, according to errors that I see when I run the script locally.